docs/en/query_language/functions/other_functions.md

```diff
@@ -394,6 +394,33 @@ FROM
 └─────────┴─────────────────────┴───────┘
 ```
 
+Please note - block size affects the result. With each new block, the `runningDifference` state is reset.
+
+``` sql
+SELECT
+    number,
+    runningDifference(number + 1) AS diff
+FROM numbers(100000)
+WHERE diff != 1
+┌─number─┬─diff─┐
+│      0 │    0 │
+└────────┴──────┘
+┌─number─┬─diff─┐
+│  65536 │    0 │
+└────────┴──────┘
+
+set max_block_size=100000 // default value is 65536!
```

Comment:
We support C++/C99 style comments, but it's strange to see them instead of SQL-style comments.

Reply:
oops. Just a typo.